unstructured/ingest/evaluate.py

```diff
@@ -165,132 +81,16 @@ def measure_text_edit_distance(
     help="Directory to save the output evaluation metrics to. Default to \
         your/working/dir/metrics/",
 )
-def measure_element_type_accuracy(
+def measure_element_type_accuracy_command(
     output_dir: str,
-    output_list: Optional[List[str]],
     source_dir: str,
+    output_list: Optional[List[str]],
     source_list: Optional[List[str]],
```

Comment:
```suggestion
    output_list: Optional[List[str]] = None,
    source_list: Optional[List[str]] = None,
```

Reply:
hm i didn't add defaults for the _command functions because they're already built into the click parameters. This change isn't necessary for functionality. Do you still have a reason for me to change it?